Certificate Transparency Fix

MrJester opened the pull request:
This is from PR #95.


Fixes cert trans module and I removed the extra edits to gitignore and adding of gitattributes.

lanmaster53 commented on modules/recon/domains-hosts/certificate_transparency.py:
```diff
@@ -7,7 +7,7 @@ class Module(BaseModule):
         'name': 'Certificiate Transparency Search',
         'author': 'Rich Warren (richard.warren@nccgroup.trust)',
-        'version': '1.1',
+        'version': '1.2',
```
Doesn't this require an index update as well?

lanmaster53 commented on modules/recon/domains-hosts/certificate_transparency.py:
```diff
@@ -20,13 +20,25 @@ def module_run(self, domains):
+                    else:
+                        self.insert_hosts(host)
+                
+                """
```
Why comment instead of just remove?

lanmaster53 commented on modules/recon/domains-hosts/certificate_transparency.py:
```diff
@@ -20,13 +20,25 @@ def module_run(self, domains):
                     self.insert_hosts(name.split('@')[1])
                 else:
                     self.insert_hosts(name)
+                """
```
Don't forget line break at the end of file.

MrJester commented:
> Also, why is the README file being moved?

I don't see where the README file is being removed.

lanmaster53 commented on README.md:
```diff
@@ -0,0 +1,5 @@
+Welcome to the Recon-ng Marketplace! The official module repository for the Recon-ng Framework.
```
Why is the README being affected by this PR?

MrJester replied:
Looks like it was being renamed earlier in the PR.

lanmaster53 replied:
ok, so if it isn't now, then why is it being seen as a different between your branch and the master?

lanmaster53 replied:
I see, it is still trying to at a README to the .github folder. Please remove that.

lanmaster53 commented on modules/recon/domains-hosts/certificate_transparency.py:
```diff
@@ -20,13 +20,16 @@ def module_run(self, domains):
-                    self.insert_hosts(name)
+                for host in cert.get('name_value').split():
+                    if '@' in host:
+                        # NOTE: This statement is UNTESTED since the last change.
```
Should it be tested first?

MrJester commented:
I've tested and the module is working fine.

On Tue, Apr 14, 2020 at 11:49 AM lanmaster53 <notifications@github.com>
wrote:

> *@lanmaster53* commented on this pull request.
> ------------------------------
>
> In modules/recon/domains-hosts/certificate_transparency.py
> <https://github.com/lanmaster53/recon-ng-marketplace/pull/112#discussion_r408243829>
> :
>
> >              for cert in resp.json():
> -                name = cert.get('name_value')
> -                if '@' in name:
> -                    self.insert_contacts(email=name)
> -                    self.insert_hosts(name.split('@')[1])
> -                else:
> -                    self.insert_hosts(name)
> +                for host in cert.get('name_value').split():
> +                    if '@' in host:
> +                        # NOTE: This statement is UNTESTED since the last change.
>
> Should it be tested first?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/lanmaster53/recon-ng-marketplace/pull/112#pullrequestreview-393061721>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABCS7ZVZTQT2QR5PSAXFYVLRMSAXPANCNFSM4MFXDIDA>
> .
>


-- 
Ryan Hays
757-869-3428
